app/src/main/java/com/jerboa/model/PostViewModel.kt

```diff
@@ -267,6 +268,62 @@ class PostViewModel(val id: Either<PostId, CommentId>) : ViewModel() {
         }
     }
 
+    fun updateBanned(personView: PersonView) {
```

Comment:
Wasn't sure what to name this one. Its only used for banning a user currently, but could potentially be used for any updates for every person in a post.